run_Wochenende.py

```diff
@@ -895,7 +1011,7 @@ def runBamtoolsFixed(stage_infile, numberMismatches):
         mmString
     ]
     bamtools_cmd1 = " ".join(filterMismatchesCmd)
-    print("bamtools cmd" + bamtools_cmd1)
+    print("Bamtools fixed cmd: " + bamtools_cmd1)
```

Comment:
BamtoolsFixed works(mm3), reports higher read counts in output as expected when compared to mm1